### ISSUE TITLE
clientSecret is never described

### ISSUE THREAD

**Issue by nathanieltagg** [AzureADQuickStarts/WebApp-OpenIDConnect-NodeJS#10]
The supporting article mentions "The application secret that you created in the app registration portal for your app."  However, the creation of this "secret" is never described.  The Azure portal does not look like the one described, and it's unclear how this secret is generated.  

Consistent names for things would be helpful for someone new to this as I am!

**Comment by lovemaths** [comment]
@nathanieltagg After you register your application on azure portal, go to the Settings or Configure page, then go to Keys section. Select the Duration, then click Save, a string will show up. This string will be the secret. You want to copy it before you leave the page, since it will be hidden once you leave the page.

**Comment by jfstephe** [comment]
@lovemaths - Sorry but your instructions are pretty hard to follow. 
A Key has 3 parts. 'Description', 'Expires' and 'Value'. Which one of those is the secret? Presumably 'value'.
If I create 3 keys with descriptions '1', '2', and '3', all with expired 'Never expires' when I click 'save'  the values are generated.
Given this scenario, what would be the 'secret'? Must I ONLY create a single key? If I create more than one do they need to have a specific description to be identified as the 'secret'?

**Comment by lovemaths** [comment]
@jfstephe Yes you are right, the secret is the 'value'. 'Description' can be anything you want, it is not used anywhere, the only purpose is to help you remember why you created this key.

If you created more than one keys, you can pick any key to use. All keys are valid.

**Comment by DaveVoyles** [comment]
Is this referring to the ```cookieEncryptionKeys``` in ```config.js```? 

I see that those are required, but am not sure of where to get them from. In the ```config.js``` file it looks like this:

```json
    // Recommended to set to true. By default we save state in express session, if this option is set to true, then
    // we encrypt state and save it in cookie instead. This option together with { session: false } allows your app
    // to be completely express session free.
    useCookieInsteadOfSession: true,
  
    // Required if `useCookieInsteadOfSession` is set to true. You can provide multiple set of key/iv pairs for key
    // rollover purpose. We always use the first set of key/iv pair to encrypt cookie, but we will try every set of
    // key/iv pair to decrypt cookie. Key can be any string of length 32, and iv can be any string of length 12.
    cookieEncryptionKeys: [ 
      { 'key': '12345678901234567890123456789012', 'iv': '123456789012' },
      { 'key': 'abcdefghijklmnopqrstuvwxyzabcdef', 'iv': 'abcdefghijkl' }
    ],
  ```

**Comment by navyasric** [comment]
Thanks for the feedback. The Key generated in the application registration is to be used as the client secret in the config. The Readme has been updated  with steps for the creation of key  to be used as client secret.

**Comment by DaveVoyles** [comment]
Thank you!
